.hlint.yaml

```diff
@@ -12,11 +12,15 @@
 - ignore: { name: Eta reduce }
 - ignore: { name: Use section }
 - ignore: { name: Use underscore }
+- ignore: { name: no recovering }
+- ignore: { name: no threadDelay }
 
 # custom rules:
 - hint: { lhs: (() <$), rhs: void }
 - hint: { lhs: return, rhs: pure }
 - hint: { lhs:  maybe mempty, rhs: foldMap }
+- hint: { name: no threadDelay, lhs: threadDelay, rhs: unrace }
+- hint: { name: no recovering, lhs: recovering, rhs: unrace }
```

Comment:
i have tried to limit those rules to package `integration`, but couldn't find a way.  limiting to modules didn't work *and* is inconvenient, as we have to remember to add every new module from `integration` here as well.

introducing the rules and then ignoring them where they are in the way seems the most robust way to do this, but this didn't work either.

i'm open to ideas and opinions, including "stupid idea, don't do this with hlint, we'll remember ourselves".